src/main/java/de/caritas/cob/consultingtypeservice/api/service/TopicServiceFacade.java

```diff
@@ -24,8 +26,18 @@ public class TopicServiceFacade {
 
   public List<TopicDTO> getAllTopics() {
     var topicEntities = topicService.getAllTopics();
-    return topicEntities.stream().map(topicConverter::toDTO).collect(
-        Collectors.toList());
+    return convert(topicEntities);
+  }
+
+  public List<TopicDTO> getAllActiveTopics() {
+    var topicEntities = topicService.getAllActiveTopics();
+    return convert(topicEntities);
+  }
+
+  private List<TopicDTO> convert(Collection<TopicEntity> topicEntities) {
```

Comment:
I would put this convert method to a Converter (to support conversion for collections)

Reply:
I thought about that, but it was late ;)
But can introduce this.

Reply:
that would be great ;)